pkg/k8shandler/collection.go

```diff
@@ -501,7 +523,10 @@ func isFluentdDaemonsetDifferent(current *apps.DaemonSet, desired *apps.DaemonSe
 
 	different := false
 
-	// TODO: populate this
+	if isImageDifference(current, desired) {
+		logrus.Infof("Fluentd image change found, updating %q", current.Name)
```

Comment:
In the logging PR from @josefkarasek I thought we had these at debug messages.  I actually prefer info since an image change is more of a major event IMO.

Reply:
No, this is a new message. IMO it should be at `info` because it is used to notify users that we need to adjust something.